EIPS/eip-7808.md

```diff
@@ -0,0 +1,65 @@
+---
+eip: 7808
```

Comment:
```suggestion
eip: 7528
```
Assigning next sequential EIP/ERC number.
EIP/ERC numbering changed to sequential from 7500 and is no longer the PR number.

Please also update the filename and the Eth Magicians discussion.